xds/internal/xdsclient/transport/transport.go

```diff
@@ -277,8 +284,11 @@ func (t *Transport) recvAggregatedDiscoveryServiceResponse(stream adsStream) (re
 	if err != nil {
 		return nil, "", "", "", fmt.Errorf("failed to read ADS response: %v", err)
 	}
-	t.logger.Infof("ADS response received, type: %v", resp.GetTypeUrl())
-	t.logger.Debugf("ADS response received: %v", pretty.ToJSON(resp))
+	if t.logger.V(perRPCVerbosityLevel) {
+		t.logger.Infof("ADS response received: %v", pretty.ToJSON(resp))
+	} else {
+		t.logger.Debugf("ADS response received for type %q, version %q, nonce %q", resp.GetTypeUrl(), resp.GetVersionInfo(), resp.GetNonce())
```

Comment:
Should the log msgs be swapped around? We want to see the whole `resp` pretty printed in debug, while only the gist in Infof()? In my experience this pretty printed JSON is pretty big :P

Reply:
Seems like this is the pattern you have stuck to in other places as well. Maybe im missing something. LMK

Reply:
I replied to the other comment.